extensions/blocks/premium-content/_inc/subscription-service/class-jwt.php

```diff
@@ -0,0 +1,441 @@
+<?php
```

Comment:
Is there a good example of how this works?

Reply:
Maybe the giant #16699, where we added the `nojimage/twitter-text-php` package. We're using it in `_inc/lib/class-jetpack-tweetstorm-helper.php`. 

In short, it would involve adding the package to Jetpack's `composer.json`, and it will be autoloaded and available to us.

Reply:
Adding this as a follow up.